src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -3006,6 +3007,17 @@ private function registerHtmlSanitizerConfiguration(array $config, ContainerBuil
             $def = $container->register($configId, HtmlSanitizerConfig::class);
 
             // Base
+            if ($sanitizerConfig['default_action'] ?? false) {
+                if (!class_exists(HtmlSanitizerAction::class)) {
```

Comment:
this is dead code because of the way the config is done at the moment: if there is a value, it must be one of the allowed cases, which are derived from the enum